src/webui/www/public/scripts/client.js

```diff
@@ -151,10 +151,9 @@ window.addEvent('load', function () {
     initializeWindows();
 
     // Show Top Toolbar is enabled by default
-    if (localStorage.getItem('show_top_toolbar') == null)
-        var showTopToolbar = true;
-    else
-        var showTopToolbar = localStorage.getItem('show_top_toolbar') == "true";
+    var showTopToolbar = true;
+    if (localStorage.getItem('show_top_toolbar') !== null)
+        showTopToolbar = localStorage.getItem('show_top_toolbar') == "true";
```

Comment:
I only replaced the `==` that jshint immediately complained about, which is when comparing against `null`, `undefined`, a boolean, or an int. You point out a valid change, however there are more instances where `==` is used against a string (a quick search yielded 121 results). I'm not opposed to going through those over time, however they should really be individually verified before changing them